readinessProbe/main.go

```diff
@@ -0,0 +1,65 @@
+// Description: This program creates a web server to verify if the managedOCS
+//              resource is ready. It is used as a readiness probe by the
+//              ocs-osd-deployer operator. For this to be set up, the following
+//              sidecar should be added to the manager CSV:
+//      - name: readinessServer
+//        command:
+//        - /readinessServer
+//        image: quay.io/rcampos/ocs-osd-deployer-readiness:latest
+//        readinessProbe:
+//          httpGet:
+//            path: /readyz
+//            port: 8081
+//          initialDelaySeconds: 5
+//          periodSeconds: 10
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis"
+	v1 "github.com/openshift/ocs-osd-deployer/api/v1alpha1"
+	"github.com/openshift/ocs-osd-deployer/readinessProbe/readiness"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
```

Comment:
I prefer if we use the controller-runtime log instead of golang log or writing directly to stdout

```suggestion
var (
	log = ctrl.Log.WithName("readiness")
)
```